src/commands/create.ts

```diff
@@ -0,0 +1,508 @@
+import colors from "colors";
+import { IFs, memfs } from "memfs";
+import { getNewProjectTemplateList } from "../requests/getTemplateList.js";
+import { Template } from "../requests/models.js";
+import git from "isomorphic-git";
+import http from "isomorphic-git/http/node/index.js";
+import { parse, stringify } from "yaml";
+import path from "path";
+import { YamlProjectConfiguration } from "../models/yamlProjectConfiguration.js";
+import { CloudProviderIdentifier } from "../models/cloudProviderIdentifier.js";
+import nativeFs from "fs";
+import { setLinkPathForProject } from "../utils/linkDatabase.js";
+import log from "loglevel";
+import { platform } from "os";
+import { GenezioCreateOptions } from "../models/commandOptions.js";
+import { debugLogger } from "../utils/logging.js";
+
+type ProjectInfo = {
+    name: string;
+    region: string;
+    type: "backend" | "frontend";
+    template: Template;
+};
+
+type FullstackProjectInfo = {
+    name: string;
+    region: string;
+    structure: "monorepo" | "multirepo";
+    backendTemplate: Template;
+    frontendTemplate: Template;
+};
+
+export async function createCommand(options: GenezioCreateOptions) {
+    const { fs } = memfs();
+    const templateList = await getNewProjectTemplateList();
+
+    const projectNameRegex = /^[a-zA-z][a-zA-Z0-9-]+$/;
+    if (!projectNameRegex.test(options.name)) {
+        throw new Error(
+            "Project name must start with a letter and contain only letters, numbers and dashes",
+        );
+    }
+
+    const projectPath = path.join(process.cwd(), options.name);
+    if (nativeFs.existsSync(projectPath)) {
+        // If the project contains only a README.md/README or a .git folder, it's safe to continue
+        const files = nativeFs.readdirSync(projectPath);
+        for (const file of files) {
+            if (file !== "README.md" && file !== "README" && file !== ".git") {
```

Comment:
i would also add LICENCE and .gitignore here

Reply:
I agree